### PR TITLE
Add protocol for sprint review (2022-09-15)

### PR DESCRIPTION
**Pull request by cdingm**

### CODE REVIEW

File: protocols/global/2022-09-15-protocol.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# Meeting Transcript 15.09.2022
+
+Review Sprint 8
+
+- present: Aaron, Florian, Leon H, Levi, Martin, Max, Michael, Timo + Coaches + Sandro
+- absent excused: Gilian, Ilijaz, Jonathan, Leon L
+
+## Feedback
+
+- change lecturer interface minigame export to json
```
Comment (delvh) [inline protocols/global/2022-09-15-protocol.md:10]:
```suggestion
- change lecturer interface minigame export filename from `*.txt` to `*.json`
```

File: protocols/global/2022-09-15-protocol.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+- change offline screen: needed --> required
```
Comment (delvh) [inline protocols/global/2022-09-15-protocol.md:11]:
```suggestion
- change offline screen text: `needed` --> `required`
```

File: protocols/global/2022-09-15-protocol.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+- 30.9
+- 10:00
+
+## Discussion Point
+
+- additional Info
+
+Result:
+
+- info on result
```
Comment (delvh) [inline protocols/global/2022-09-15-protocol.md:28]:
```suggestion
```

Comment (cdingm) [reply protocols/global/2022-09-15-protocol.md:28]: oops

**Review by delvh — APPROVED** [review]

**Pull request merged**